src/d3a/setup/default_5.py

```diff
@@ -29,105 +29,118 @@ def get_setup(config):
     area = Area(
         'Grid',
         [
-            Area(
```

Comment:
I would strongly suggest to create a new setup file and do not use default_5 file just for this. The reason is that this file is used heavily for testing (it is the 5-house microgrid on the UI) and I would leave it as is in order for the people that test it to not be confused by the addition of the Street hierarchy.